backend/accounts/views.py

```diff
@@ -39,6 +40,32 @@
 from accounts.verification import sendEmailVerification, sendSMSVerification
 
 
+def image_upload_helper(request, cls, keyword, field):
+    """
+    Given a Model object, save the uploaded image to the image field specified
+    in the argument.
+    Returns a response that can be given to the end user.
```

Comment:
Can we specify in the comments the type of `field`, `cls` (what is cls?), and `keyword`, and how they will affect the end result of this upload helper?